src/plugins/ima/index.js

```diff
@@ -38,11 +36,11 @@ export default function imaPlugin(player, playerOptions) {
   if (Object.keys(playerOptions.ads).length > 0 && typeof player.ima === 'object') {
     if (playerOptions.ads.adsInPlaylist === 'first-video') {
       player.one(PLAYER_EVENT.SOURCE_CHANGED, () => {
-        player.ima.playAd();
+        player.ima.playAdBreak();
```

Comment:
what is the difference between these two methods? just curious or is it just a rename?